src/serviceLauncher.ts

```diff
@@ -36,15 +36,25 @@ function main() {
                 }
             )
         } else if (service.type == 'ceramic') {
+            // Removing port due to https://github.com/ceramicnetwork/js-ceramic/issues/1681
+            // and verbose logging
             subprocess = exec(
                 `node node_modules/@ceramicnetwork/cli/bin/ceramic daemon \
-                    --verbose \
```

Comment:
What messages do we lose by turning off verbose logging?  Unless we have a strong reason to do so, I'd rather leave as much log output as possible right now so we have more information to use while debugging.

Reply:
Agreed.

My thought was to get the tests running first and then figuring out why verbose logging is causing failures. I can add a TODO to this effect if that helps.

I'm hoping not to hold the release up if it takes us a while to figure out the failures.